frontend/src/components/teacher/student-management/classroom-summary/ClassroomPopover.tsx

```diff
@@ -116,6 +116,13 @@ const ClassroomPopover = ({
         body={
           'Are you sure you want to archive this classroom? You can find your archived classrooms in the "Archived" tab.'
         }
+        customBodyText={
+          <Text color="red.200" pt={2} textStyle="paragraph">
+            Note that all <strong>upcoming</strong> assessments will be{" "}
```

Comment:
linter added these spaces

Reply:
happens lol

Reply:
also is using the `strong` tag and `u` considered best practice? IIRC those tags are deprecated or something in favour of using CSS but not really sure

Reply:
To be honest, not too sure we need the emphasis here. I think red text is probably enough

Reply:
fair